### ISSUE TITLE
Accessibility: ORCA screen reader is not announcing name property

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212319]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Windows 11 Enterprise 23H2 (OS build 22631.3296)

Copied from https://github.com/microsoft/vscode-azureapicenter/issues/122

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #ORCA; #Linux; #FTP;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Screen reader: ORCA

### Repro Steps:

1. Start ORCA screen reader.
2. Create an API center in azure portal.
3. Open Visual studio Code.
4. Tab till Chat option in the left side panel and press enter.
5. Type @terminal in chat box and hit ENTER key.
6. TAB to "What are some popular front end framework for web development" and "Explain the suggested command" buttons.
7. Observe the issue.

### Actual Result:
ORCA screen reader is not announcing name property of "What are some popular front end framework for web development" and "Explain the suggested command" buttons.

### Expected Result:
ORCA screen reader should be announcing name property of "What are some popular front end framework for web development" and "Explain the suggested command" buttons.

### User Impact:
Screen reader users will not be able to get the information about the controls and will miss the information.

### Attachment:
![122_A11y_VS Code extension for API Center_Copilot Chat_ScreenReader](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/378672db-89cb-41b1-a0e8-c876eb1891ee)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.89.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by formulahendry** [comment]
Tested on latest 1.89.0, still have this issue.

**Comment by meganrogge** [comment]
isn't this a duplicate of https://github.com/microsoft/vscode/issues/212323?

if not, what happens when you tab to those?

can you reproduce on NVDA? if not, this is an issue with ORCA, and should be opened for them instead of us.

**Comment by formulahendry** [comment]
> isn't this a duplicate of #212323?
> 
> if not, what happens when you tab to those?
> 
> can you reproduce on NVDA? if not, this is an issue with ORCA, and should be opened for them instead of us.

@PoojaNamde Please confirm.
